src/runtime/auto-prompt-manager-test.js

```diff
@@ -4753,6 +4753,19 @@ describes.realWin('AutoPromptManager', (env) => {
       expect(isEligible).to.be.true;
     });
 
+    it('Orchestration with finite eligibility defaults to 1 eligible completion', async () => {
```

Comment:
Not requesting a change since all the other tests are like this too, but I think it would be better to keep act / assert blocks separated.
go/unit-testing-practices#structure

Reply:
Updated the tests of in this suite for helper functions.